source/administration/changelog.md

```diff
@@ -6,7 +6,129 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ClusterSettings`` in ``config.json``:
+     - Added ``EnableGossipCompression``, to disable compression in the Gossip protocol. By default the value of the setting is ``true``, which is the existing default. This is done to maintain compatibility with old servers in a cluster. Once all servers in a cluster are upgraded, it is recommended to disable this setting for better performance.
+ - Under ``SqlSettings`` in ``config.json``:
+     - Added ``ConnMaxIdleTimeMilliseconds``, to allow controlling the maximum time a database connection can remain idle. The default value is set to 5 minutes.
+ - Under ``TeamSettings`` in ``config.json``:
+     - Added ``EnableCustomUserStatuses``, to allow users to set descriptive status messages and optional status emoji that are visible to all users.
+    
+### Go Version
+ - 5.33 is built with Go ``1.15.5``.
+
+### Open Source Components
+ - Added ``types/react-overlays``, ``crypto-browserify``, ``process`` and ``stream-browserify``, and removed ``node-semver`` from https://github.com/mattermost/mattermost-webapp.
+ - Removed ``isomorphic-fetch`` from https://github.com/mattermost/mattermost-redux.
+
+### API Changes
+ - Added a new ``GET /{team_id}/threads/{thread_id}`` API method for retrieving single threads.
+ - Added a new ``/exports`` API endpoint to generate and manage export files.
+ - Added a new ``/users/{user_id}/teams/{team_id}/threads/mention_counts`` API endpoint.
+ - Added a new ``GET /api/v4/cloud/subscription/stats`` API endpoint.
+ - Added a new ``GET /api/v4/cloud/subscription/limitreached/invite`` API endpoint.
+ - Added new ``PUT /api/v4/users/<id>/status/custom``, ``DELETE /api/v4/users/<id>/status/custom``, and ``DELETE /api/v4/users/<id>/status/custom/recent`` API endpoints.
+ - The ``/api/v4/users/me/auth`` API endpoint can no longer be used to change passwords. This was a hidden feature that was not documented, but was nevertheless possible. We are just removing the hidden feature.
+ - Updated ``/users/{user_id}/teams/{team_id}/threads`` API to support ``unread=true`` query parameter.
+ - ``/api/v4/users/{user_id}/teams/{team_id}/threads`` API endpoint now accepts "before" and "after" parameters instead of a page index.
+ - Removed session required restriction from ``GET api/v4/subscription/stats`` API endpoint.
+
+### Websocket Event Changes
+ - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD-based distributions.
+ - ``UserUpdate`` WebSocket Event is now broadcast by two more APIs, ``plugin.UpdateUser`` and ``ConvertBotToUser``.
+
+### Known Issues
```

Comment:
Incredibly minor nit: there's inconsistency with where we place sentence-ending periods for known issues. In some sections, they're added at the end of the natural sentence, and in others, the period is added after the Jira ticket link.